index.ts

```diff
@@ -1 +1,2 @@
-export * from "./src/ExampleComponent";
+// export * from "./src/ExampleComponent";
+export * from "./src/SignInComponent";
```

Comment:
I wonder if there's a different name we could use for this component? `SignInComponent` sounds quite generic as is. Maybe `PremiumSubscriberAppBanner` or something similar?